app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/InsufficientConfidenceLevelController.scala

```diff
@@ -73,7 +72,7 @@ class InsufficientConfidenceLevelController @Inject()(
   )(implicit request: RequestWithSessionData[_]): Future[Result] =
     request.sessionData.flatMap(_.journeyStatus) match {
       case Some(i: IndividualWithInsufficientConfidenceLevel) => f(i)
```

Comment:
As discussed, `IndividualWithInsufficientConfidenceLevel` seems to represent a user whereas what we are attempting to represent is an actual type of user journey. As such, this should be renamed to reflect that. A possible name to represent this part of the journey could be "DetermineAffinityGroup"

Reply:
I renamed it to `TryingToGetIndividualsFootprint` - someone has this journey status when they have CL<200 and we can't find a NINO or SAUTR. We take them through a journey where we ask if they have a NINO or SAUTR and subsequently if they are trying to register an individual or trust